src/components/Fieldset/DynamicFieldset/DynamicFieldset.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import React, {useState} from 'react';
+import {StoryObj, Meta} from '@storybook/react';
+
+import {DynamicFieldset} from './index';
+import {Field, FieldSelector} from '~/components';
+import markdownNotes from './DynamicFieldset.md';
+import {Button, Chip, Input} from '~/components';
+import {Add, Language, MoreVert} from '~/icons';
+
+const meta: Meta<typeof DynamicFieldset> = {
+    title: 'Components/DynamicFieldset',
```

Comment:
I think it will be better to put `DynamicFieldset` in the same "folder" than the `Fieldset` component in storybook to keep both closer